rails/blueprints/simple_form/template.rb

```diff
@@ -0,0 +1,19 @@
+def apply_self!
+  if yes?("Would you like to install Simple Form?")
+    gem "simple_form"
+    run "bundle install"
+
+    if yes?("Do you use Bootstrap?")
+      generate "simple_form:install --bootstrap"
+    elsif yes?("Do you use Tailwind?")
```

Comment:
might be a dumb question, but i'm assuming there's no case where someone would want to use both tailwind and bootstrap, correct?

Reply:
Good question but I don't think that would be the case because this is for greenfield apps and including both would probably be a poor design decision.

Reply:
That's what I assumed, but figured I'd check!